src/Integration/Suppression/ManagedMD5.cs

```diff
@@ -0,0 +1,461 @@
+﻿/* Copyright (C) David Anson
+ * https://dlaa.me/blog/post/9380245
```

Comment:
What, if anything, needs to go into the `THIRD-PARTY-NOTICES` file?

Reply:
I don't see a copyright in his project/site that I can point to. 
How about this:
```
License notice for MD5 Algorithm
-----------------------------------------------------
Copyright (C) David Anson 
https://dlaa.me/blog/post/9380245
```

Reply:
Seems ok. What about the RSA license notice from the comments in lines 10-44?